package.json

```diff
@@ -9,7 +9,9 @@
   "scripts": {
     "prebuild": "rm -rf lib dist .cache",
     "build": "npm-run-all build:pkg --parallel build:src:* --parallel build:pages:* build:themeable",
-    "lint": "eslint --ignore-path .gitignore --ext ts,tsx,js .",
+    "lint": "npm-run-all --parallel lint:*",
```

Comment:
should we also add to `lint-staged` config? https://github.com/cloudscape-design/chat-components/blob/main/package.json#L110